SECURITY.md

```diff
@@ -19,5 +19,4 @@ After validating & discussing scope of security vulnerability, we will set a tim
 
 Once affected versions are patched you may report security issue to any Node.js security vulnerability database. A few which we have worked with in past are listed below.
 
-- [NPM](https://www.npmjs.com/advisories/report)
```

Comment:
Seems it was replaced with the builtin GitHub advisories. Can you replace it with that one? https://github.com/sequelize/sequelize/security/advisories/new

Reply:
We should be the ones to publish those advisories, should we rephrase to "You can either publish an advisory, or we will publish our own using GitHub advisories" or something like that?

Reply:
The link from GitHub does actually create a draft advisory that we can approve and publish. So maybe we can recommend the one from GitHub and have Snyk as a backup in case people do not have a GitHub account?

Reply:
I know but so far, either the person reporting the issue was a vulnerability researcher that had access to their own advisories, or they sent us a pretty barebones advisories that wasn't really usable. I think we should just tell them to either publish to their own advisories if they're a researcher, or let us do it via gh

Reply:
I updated the commit to contain the security advisory link for Github. Thoughts?

Reply:
I think it's good. Let's keep a reference to CONTACT.md in there as well in case people do not have or want to create a GitHub account.

Reply:
done, is that looking good?